python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
 
-Estimation of a quantile by Wilks' method
------------------------------------------
+Estimation of a quantile upper bound by Wilks' method
+-----------------------------------------------------
 
-Let us denote
-:math:`\underline{Y} = h\left( \vect{X},\vect{d} \right) = \left( Y^1,\ldots,Y^{n_Y} \right)`,
-where :math:`\vect{X}= \left( X^1,\ldots,X^{n_X} \right)` is a random
-vector, and :math:`\vect{d}` a deterministic vector. We seek here to
-evaluate, using the probability distribution of the random vector
-:math:`\vect{X}`, the :math:`\alpha`-quantile :math:`q_{Y^i}(\alpha)` of
-:math:`Y^i`, where :math:`\alpha \in (0, 1)`:
+We consider a random variable :math:`X` of dimension 1 and its unknown quantile :math:`x_{\alpha}` of order :math:`\alpha`.
+We seek to evaluate an upper bound of :math:`x_{\alpha}` with a confidence greater than :math:`\beta`, using a given order statistics.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th maximum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum. We have:
 
 .. math::
 
-   \begin{aligned}
-       \Prob{ Y^i \leq q_{Y^i}(\alpha)} = \alpha
-     \end{aligned}
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)}
 
-If we have a sample
-:math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of :math:`N`
-independent samples of the random vector :math:`\vect{X}`,
-:math:`q_{Y^i}(\alpha)` can be estimated as follows:
 
--  the sample :math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of
-   vector :math:`\vect{X}` is first transformed to a sample
-   :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` of the variable
-   :math:`Y^i`, using :math:`\underline{y} = h(\vect{x}_i,\vect{d})`,
+Estimation of the order statistics that will provide an upper bound to :math:`x_{\alpha}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  the sample :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` is then placed
-   in ascending order, which gives the sample
-   :math:`\left\{ y^{(1)},\ldots,y^{(N)} \right\}`,
+We assume that we have a sample of size :math:`\sampleSize` of :math:`X`, denoted by :math:`(x_1, \dots, x_\sampleSize)`.
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`\sampleSize`, we seek for the smallest :math:`k` such that:
+
+.. math::
+    :label: EqOrderStat
 
--  this empirical estimation of the quantile is then calculated by the
-   formula:
+    \Prob{X_{(k)} \geq x_{\alpha}} \geq \beta
```

Comment:
I believe that exchanging the expressions makes it clearer that $X_{(k)}$ is an _upper_ bound of the quantile.
```suggestion
    \Prob{x_{\alpha} \leq X_{(k)}} \geq \beta
```

Reply:
yes